tests/UnifiedSpecTests/UnifiedTestCase.php

```diff
@@ -25,20 +25,15 @@
  */
 final class UnifiedTestCase implements IteratorAggregate
 {
-    /** @var stdClass */
-    private $test;
+    private stdClass $test;
```

Comment:
Interesting that rector left `stdClass` here but changed it to `object` (I assume) in `RunOnRequirement.php`.